src/lib/types/proposal.ts

```diff
@@ -28,4 +28,5 @@ export interface Proposal {
   resolvedHeight: Option<number | null>;
   type: ProposalType;
   proposer: Option<Addr>;
+  isExpedited: Option<boolean | null>;
```

Comment:
I think its just boolean

Reply:
The generated type is `boolean | null | undefined`